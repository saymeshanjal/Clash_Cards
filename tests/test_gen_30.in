99
16 389 3
4 74 3
6 175 1
2 58 4
6 151 5
14 308 5
16 346 4
10 61 1
2 244 2
4 129 2
12 312 3
10 145 2
4 92 4
10 268 2
10 62 5
14 340 4
2 44 2
4 72 3
16 344 5
6 185 4
12 87 2
12 35 2
16 125 1
2 257 1
8 125 3
14 28 1
12 196 5
14 213 2
12 141 2
2 359 4
10 28 3
4 289 1
2 16 1
6 88 3
4 211 1
12 41 4
14 334 2
8 141 1
4 370 4
16 18 3
14 366 1
12 176 5
16 344 3
14 114 3
14 155 4
10 129 3
14 327 2
12 395 5
4 356 5
6 107 5
6 281 2
14 118 5
8 5 4
2 19 5
12 66 1
12 174 5
16 21 2
2 154 4
12 339 2
12 317 4
14 47 1
14 309 4
14 240 2
14 160 1
4 205 2
16 42 2
14 210 5
10 146 1
14 343 5
8 20 4
10 347 3
14 41 2
14 128 4
10 6 4
14 239 1
4 182 3
8 316 3
6 217 1
8 183 2
16 170 2
2 238 4
2 364 1
12 123 1
6 15 4
16 365 1
4 87 3
14 58 2
10 349 1
12 264 1
12 96 3
4 160 2
4 9 3
6 350 1
8 273 4
6 192 4
4 362 3
16 24 2
4 140 1
10 57 5
