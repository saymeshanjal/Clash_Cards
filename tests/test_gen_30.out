6307 6563 0
1 5 0
10 10 0
0 2 0
1 19 0
987 2445 0
3809 5406 3655
126 126 0
0 1 1
1 2 3
430 494 0
76 77 99
0 5 1
76 77 99
45 207 0
910 1520 1002
0 1 1
1 5 0
4248 8622 0
1 14 5
286 287 351
126 170 628
6435 6435 0
1 1 0
27 43 0
1615 1817 0
219 705 0
1078 1079 1275
286 287 351
0 2 0
66 186 0
3 3 0
1 1 0
6 14 0
3 3 0
69 498 357
1078 1079 1275
35 35 0
0 5 1
1522 11348 0
1716 1716 0
219 705 0
6307 6563 0
1652 1780 0
910 1520 1002
110 142 0
1078 1079 1275
219 705 0
0 6 0
1 19 0
5 6 9
987 2445 0
0 69 1
0 2 0
462 462 0
219 705 0
339 1760 10771
0 2 0
286 287 351
208 441 275
1710 1722 0
910 1520 1002
1078 1079 1275
1716 1716 0
1 2 3
378 1003 11489
987 2445 0
126 126 0
987 2445 0
1 50 19
110 142 0
298 461 2673
910 1520 1002
0 235 17
1716 1716 0
1 5 0
27 43 0
10 10 0
20 21 29
4081 4082 4707
0 2 0
1 1 0
462 462 0
1 14 5
6435 6435 0
1 5 0
1029 1036 1367
126 126 0
462 462 0
430 494 0
1 2 3
1 5 0
10 10 0
8 42 20
1 14 5
1 5 0
346 1528 10996
3 3 0
45 207 0
